[SC-434] Chore: Update Readmes (#527)
InverterNetwork/inverter-contracts: README.md

## Usage

Common tasks are executed through a `Makefile`. The most common commands are:
* `make build` to compile the project.
* `make test` to run the test suite.

>   * Note: _Some of our tests  require a working Sepolia RPC URL, as we test certain contracts via fork testing. We implemented fallbacks in these cases in the code directly, so they should work even without any RPC set in the environment. If that does not work, please set a working one via `export SEPOLIA_RPC_URL=https://rpc-url-here`._

Reviewer: ```suggestion
  * Note: _Some of our tests  require a working Sepolia RPC URL, as we test certain contracts via fork testing. We implemented fallbacks for these particular test cases in the code directly, so they will work even without any RPC set in the environment. In the unlikely case the tests do not work without RPC, please set a working RPC via `export SEPOLIA_RPC_URL=https://rpc-url-here`._
```

Author: Addressed via next commit.
